### PR TITLE
Add customers

### PR DESCRIPTION
**Pull request by jbrad**

### CODE REVIEW

**Comment by BrendanVoight** [comment]
A comment

**Comment by VyNgo97** [comment]
what did he want us to do again?

**Comment by BryceValley** [comment]
A superior comment

File: spikes-swagger.yaml [hunk 1]
```diff
@@ -11,6 +11,140 @@ servers:
...
+      parameters:
+        - name: legacyCustomerNumber
+          in: query
+          description: Legacy identifier for a customer
+          required: false
+          schema:
+            type: number
+      responses:
+        "200":
+          description: A collection of customers
```
Comment (arogers325) [inline spikes-swagger.yaml]: ❤️

File: spikes-swagger.yaml [hunk 2]
```diff
@@ -140,6 +274,49 @@ paths:
...
+          type: string
+        lastName:
+          type: string
+        legacyCustomerNumber:
+          type: number
+          format: integer
+      example:
+        customerId: A6083CAA-54E4-4744-B23B-374056F1B5D7
+        firstName: "Ron"
+        lastName: "Tart"
```
Comment (VyNgo97) [inline spikes-swagger.yaml]: should be "farts"

File: spikes-swagger.yaml [hunk 3]
```diff
@@ -11,6 +11,140 @@ servers:
   - url: https://api.drake.drake/
     description: Prod Environment
 paths:
+  /customers:
```
Comment (BryceValley) [inline spikes-swagger.yaml]: This is where the code starts

File: spikes-swagger.yaml [hunk 4]
```diff
@@ -11,6 +11,140 @@ servers:
...
+    get:
```
Comment (m4gIcIaN) [inline spikes-swagger.yaml]: Let's get this bread

File: spikes-swagger.yaml [hunk 5]
```diff
@@ -11,6 +11,140 @@ servers:
...
+      description: Returns a collection of customers
+      parameters:
```
Comment (luciagarrett) [inline spikes-swagger.yaml]: rpg in the comment section

File: spikes-swagger.yaml (hunk 1 above)
Comment (jbrad) [reply spikes-swagger.yaml]: Thanks @arogers325!

File: spikes-swagger.yaml (hunk 2 above)
Comment (jbrad) [reply spikes-swagger.yaml]: 🍑

File: spikes-swagger.yaml (hunk 4 above)
Comment (BryceValley) [reply spikes-swagger.yaml]: Yeet this wheat

File: spikes-swagger.yaml [hunk 6]
```diff
@@ -11,6 +11,140 @@ servers:
...
+          schema:
+            type: number
+      responses:
+        "200":
+          description: A collection of customers
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Customers"
+        "401":
```
Comment (JacksonMeyer) [inline spikes-swagger.yaml]: comment

**Comment by jbrad** [comment]
:shipit:

**Comment by Matthew-Starcevic** [comment]
I'm trapped inside this computer, please help.

**Comment by MaddieButler** [comment]
Who's Marty?

File: spikes-swagger.yaml [hunk 7]
```diff
@@ -11,6 +11,140 @@ servers:
...
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "204":
+          description: Customer successfully updated
+        "404":
+          description: Customer not found
+          content:
+            application/json:
```
Comment (patrickheger4867) [inline spikes-swagger.yaml]: json, the son of j.

**Comment by VyNgo97** [comment]
> Who's Marty?

https://lmgtfy.com/?q=marty+mcfly

**Pull request merged**